lib/chef/provider/aws_cache_subnet_group.rb

```diff
@@ -0,0 +1,56 @@
+require 'chef/provisioning/aws_driver/aws_provider'
+
+class Chef::Provider::AwsCacheSubnetGroup < Chef::Provisioning::AWSDriver::AWSProvider
+
+  protected
+
+  def create_aws_object
+    converge_by "create new Elasticache Subnet Group #{new_resource.name} in #{region}" do
+      new_resource.driver.elasticache.create_cache_subnet_group(desired_options)
```

Comment:
I would make a getter for `new_resource.driver.elasticache` so you don't have to type so much :smile: